README.md

```diff
@@ -20,9 +20,83 @@ Link to the extension [source code](https://github.com/Susmita-Dey/Moody).
 Moody is licensed under the MIT License - see the [LICENSE](LICENSE) file for details.
 
 ## Open Source Programs
+# 🎉Hacktoberfest 2022🎉
+
+<b>(◔◡◔)&nbsp;
+<img src="https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg">
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat&logo=github)](https://github.com/Susmita-Dey/Moody-website)
+[![contributions welcome](https://img.shields.io/static/v1.svg?label=Contributions&message=Welcome&color=0059b3)](https://github.com/Susmita-Dey/Moody-website)
+[![Hacktoberfest-22](https://img.shields.io/static/v1.svg?label=Hacktoberfest-22&message=accepted&color=red)](https://hacktoberfest.com/) 
+[![Open Source Love](https://img.shields.io/badge/Open%20Source-%F0%9F%A4%8D-Green)](https://hacktoberfest.com/) 
+</b>
+<br>
  <a href="https://hacktoberfest.com/">
  <img width="1200" alt="Email Banners-Dark" src="https://user-images.githubusercontent.com/79099734/189589410-ca17afb8-5855-4316-918a-054f27594809.png">
  </a>
+ 
+ - Guys, as the name of repository suggests, this is an <b>open-source contribution</b> opportunity for everyone for [hacktoberfest](https://hacktoberfest.digitalocean.com/) 2022. 
+- So, you can add your code and pull your request. 
+- If it fulfills criteria, I will definitely merge it. 
+- Hurry up guys...!! You are only few steps away to complete your Hacktoberfest challenge. Make 4 pull requests and complete your hacktoberfest challenge and get a chance to win T-shirt. 
+- It is for only 1 month.
+
+# 📚 Contribution rules
+- Do NOT remove other content.
+- Add your name to the <b>CONTRIBUTORS.md</b> file
+- Add me as reviewer
+- Try to keep pull requests small to minimize merge conflicts.</li>
+
+# 🤩🤗 Getting Started
+    
+- Fork this repo (button on top)
+- Clone on your local machine
+
+```
+git clone https://github.com/Susmita-Dey/Moody-website.git
+```
+- Navigate to project directory.
+```
+cd Moody-website
+```
+
+- Create a new Branch
+
+```markdown
+git checkout -b my-new-branch
+```
+- Add your contribution
+```
+git add .
+```
+- Commit your changes.
+
+```markdown
+git commit -m "Relevant message"
+```
+- Then push 
+```
+git push origin my-new-branch
+```
+
+- Create a new pull request from your forked repository
+
+# ❓ FAQs (Frequently Asked Questions)
+
+- Who all can contribute?
+  - Anyone with a github account and who is signed up for
+[hacktoberfest](https://hacktoberfest.digitalocean.com/) 2022 :)
```

Comment:
Apart from hacktoberfest, anyone can contribute. This repository is open for all.

Reply:
> Apart from hacktoberfest, anyone can contribute. This repository is open for all.

Okay lemme change it